contracts/BytesLib.sol

```diff
@@ -287,11 +287,11 @@ library BytesLib {
     }
 
     function toUint(bytes _bytes, uint _start) internal  pure returns (uint256) {
-        require(_bytes.length >= (_start + 32));
+        require(_bytes.length >= (_start + 4));
         uint256 tempUint;
 
         assembly {
-            tempUint := mload(add(add(_bytes, 0x20), _start))
+            tempUint := mload(add(add(_bytes, 0x04), _start))
```

Comment:
why original implementation is `0x20` ??

Reply:
0x20 = 32..........

Reply:
ya!w